Assets/MRTK/Tools/MSBuild/MSBuildMRTKTemplates/SDKProjectTemplate.csproj.template

```diff
@@ -150,7 +150,7 @@
   </Target>
 
   <Target Name="PostBuild" AfterTargets="PostBuildEvent">
-    <Exec Command="xcopy /y $(TargetDir)$(TargetName).dll $(PublishOutputPath)$(UnityConfiguration)\$(UnityPlatform)\&#xD;&#xA;xcopy /y $(TargetDir)$(TargetName).pdb $(PublishOutputPath)$(UnityConfiguration)\$(UnityPlatform)\&#xD;&#xA;xcopy /y $(TargetDir)$(TargetName).xml $(PublishOutputPath)$(UnityConfiguration)\$(UnityPlatform)\" />
+    <Exec Command="xcopy /y $(TargetDir)$(TargetName).dll $(PublishOutputPath)$(UnityConfiguration)\$(UnityPlatform)\&#xD;&#xA;xcopy /y $(TargetDir)$(TargetName).pdb $(PublishOutputPath)$(UnityConfiguration)\$(UnityPlatform)\&#xD;&#xA;xcopy /y $(TargetDir)$(TargetName).xml $(PublishOutputPath)$(UnityConfiguration)\$(UnityPlatform)\ || set errorlevel=0" />
```

Comment:
Does this set the error level just for the last command, or for all of them?

Reply:
Just did an experiment locally and it seems that the error level applies to all three xcopy commands, but I do not expect it to be a problem as xcopy most likely fails only due to file not found issue here, and if that were to be the case there will be error stopping the process prior to the post build event.